scripts/errcheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Check gofmt
+echo "==> Checking AWS provider for unchecked errors..."
+echo "==> NOTE: at this time we only look for uncheck errors in the AWS package"
+
+if ! which errcheck > /dev/null; then
+    echo "==> Installing errcheck..."
+    go get -u github.com/kisielk/errcheck
+fi
+
+err_files=$(errcheck -ignoretests -ignore 'github.com/hashicorp/terraform/helper/schema:Set,bytes:.*,os:.*,io:.*' ./builtin/providers/aws/...)
```

Comment:
Can this be split across multiple lines (maybe with `\`?) so that its easy to modify this going forward?

Can we use multiple `-ignore` flags instead? (not sure if possible from errcheck)